PSRabbitMq/Register-RabbitMqEvent.ps1

```diff
@@ -153,15 +156,17 @@
 
         [string]$ListenerJobName, 
 
-        $ActionData
+        $ActionData,
+        
+        [int16]$Port
```

Comment:
Maybe we want to set a default value for this parameter to be backwards-compatible and for the user’s convenience.